scripting/get5/matchconfig.sp

```diff
@@ -658,66 +663,82 @@ static bool LoadMatchFromJson(JSON_Object json) {
   return true;
 }
 
-static void LoadTeamDataJson(JSON_Object json, Get5Team matchTeam) {
-  GetTeamPlayers(matchTeam).Clear();
-  GetTeamCoaches(matchTeam).Clear();
-
+static bool LoadTeamDataJson(JSON_Object json, Get5Team matchTeam, bool loadFromMatchConfig, bool allowFromFile = true) {
   char fromfile[PLATFORM_MAX_PATH];
-  json_object_get_string_safe(json, "fromfile", fromfile, sizeof(fromfile));
-
+  if (allowFromFile) {
+    json_object_get_string_safe(json, "fromfile", fromfile, sizeof(fromfile));
+  }
   if (StrEqual(fromfile, "")) {
+    GetTeamPlayers(matchTeam).Clear();
+    GetTeamCoaches(matchTeam).Clear();
     // TODO: this needs to support both an array and a dictionary
     // For now, it only supports an array
+    json_object_get_string_safe(json, "name", g_TeamNames[matchTeam], MAX_CVAR_LENGTH,  matchTeam == Get5Team_Spec ? CONFIG_SPECTATORSNAME_DEFAULT :  "");
```

Comment:
The autoformatter will probably pick this up (when I fix it, sorry it's been so long) but shouldn't be an issue.

Reply:
Yeah I'll run it before I merge.